src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.AbstractFigure;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES_BOUND = 4;
+    private static final int MAX_VALUE_BOUND = 100;
+    private static final String DEFAULT_COLOR = "white";
+    private static final double DEFAULT_RADIUS = 10.0;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public AbstractFigure getRandomFigure() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomFigureNumber = random.nextInt(NUMBER_OF_FIGURES_BOUND);
+        if (randomFigureNumber == 0) {
```

Comment:
Let’s  use switch-cases

Reply:
Is it good that I divided the classes into packages?

Reply:
It’s not good it is great 🔥 but give names of packages in the singular😉